source/player/Player.hx

```diff
@@ -3,47 +3,50 @@ package player;
 import flixel.FlxG;
 import flixel.FlxSprite;
 
-class Player extends FlxSprite
-{
-	public static var SPEEDS(default, never):Array<Int> = [0, 50, 100];
+class Player extends FlxSprite {
+	public static var SPEEDS(default, never):Array<Int> = [0, 50, 100, 150, 200];
 
-	public function new(X:Float = 0, Y:Float = 0)
-	{
+	public function new(X:Float = 0, Y:Float = 0) {
 		super(X, Y);
 
-		acceleration.y = 300;
+		acceleration.y = 330;
 	}
 
-	override public function update(elapsed:Float)
-	{
+	override public function update(elapsed:Float) {
 		setSpeed();
 		jump();
 
-		if (!isOnScreen())
-		{
+		if (!isOnScreen()) {
 			kill();
 		}
 		super.update(elapsed);
 	}
 
-	private function setSpeed()
-	{
-		var speedSelector = 1;
-		FlxG.keys.pressed.LEFT ? speedSelector-- : null;
-		FlxG.keys.pressed.RIGHT ? speedSelector++ : null;
-		velocity.x = SPEEDS[speedSelector];
+	private function setSpeed() {
+		var speedSelector = 2;
+		if (FlxG.keys.pressed.LEFT && x > 20) {
+			velocity.x = -SPEEDS[speedSelector] * 1.4;
+		} else if (FlxG.keys.justReleased.LEFT) {
+			velocity.x = 0;
+		}
+
+		if (FlxG.keys.pressed.RIGHT && x < FlxG.width - width) {
+			velocity.x = SPEEDS[speedSelector];
+		} else if (FlxG.keys.pressed.RIGHT && x > (FlxG.width - width)) {
+			velocity.x = 0;
+		} else if (FlxG.keys.justReleased.RIGHT) {
+			velocity.x = 0;
+		}
 	}
 
-	private function jump()
-	{
-		if (FlxG.keys.justPressed.UP)
-		{
-			velocity.y = -200;
+	private function jump() {
+		if (FlxG.keys.justPressed.UP) {
+			velocity.y = -220;
 		}
 	}
 
-	override function kill()
-	{
-		reset(FlxG.width / 2, FlxG.height / 2);
+	// could be changed when the life system is implemented
+	override function kill() {
+		reset(FlxG.width / 4, FlxG.height / 4);
 	}
```

Comment:
Life system is already in place, you'll want to clean up this comment and make sure your stuff plays nice with the new systems when you pull in the updated `main` branch.